tests/console/docker_image.pm

```diff
@@ -25,6 +25,10 @@ use containers::common;
 use suse_container_urls 'get_suse_container_urls';
 use version_utils qw(is_sle is_opensuse is_tumbleweed is_leap);
 
+sub isPublicCloud {
```

Comment:
maybe it would be a good idea to make this more generic and add it to version_utils.pm
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/version_utils.pm#L28
So we can use it in any module by doing `if (is_public_cloud)`

Reply:
Done